### PR TITLE
veBAL OTC: Additional helper fns and renaming

### PR DESCRIPTION
**Pull request by thomas-waite**
# Additional helper fns and renaming

### CODE REVIEW

File: proposals/dao/tip_vebalotc.ts [hunk 1]
```diff
@@ -90,23 +90,25 @@ const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts,
...
   ).to.be.equal(vebalOtcBuyer.toLowerCase());
 
   // Check that OTC Buyer can transfer proxy ownership
-  await contracts.veBoostManagerProxy.connect(otcBuyerSigner).changeAdmin(addresses.feiDAOTimelock);
+  await contracts.balancerGaugeStakerProxy.connect(otcBuyerSigner).changeAdmin(addresses.feiDAOTimelock);
   expect(
-    '0x' + (await ethers.provider.getStorageAt(addresses.veBoostManagerProxy, PROXY_ADMIN_STORAGE_SLOT)).slice(26)
+    '0x' + (await ethers.provider.getStorageAt(addresses.balancerGaugeStakerProxy, PROXY_ADMIN_STORAGE_SLOT)).slice(26)
   ).to.be.equal(addresses.feiDAOTimelock.toLowerCase());
 };
```
Comment (miguelmtzinf) [inline proposals/dao/tip_vebalotc.ts:130]: Lets check the implementation too.
`bytes32 internal constant _IMPLEMENTATION_SLOT = 0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`

**Review by miguelmtzinf — CHANGES_REQUESTED** [review]
Missing tests for withdrawETH and checking the `onlyOwner`

File: proposals/dao/tip_vebalotc.ts (hunk 1 above)
Comment (thomas-waite) [reply proposals/dao/tip_vebalotc.ts:130]: Have added a check that the implementation address stored before the upgrade is the old implementation contract, and the one after the upgrade is the new implementation contract

**Comment by thomas-waite** [comment]
> Missing tests for withdrawETH and checking the `onlyOwner`

Added. Was going to do a seperate PR adding tests for all remaining functions (they are simply pass throughs so should be fine) and include there, have added these two specific ones here

**Pull request merged**